tasmota/tasmota_xsns_sensor/xsns_18_pms5003.ino

```diff
@@ -171,6 +184,59 @@ bool PmsReadData(void)
   return true;
 }
 
+bool ZH03ReadDataPassive() // process the passive mode response of the ZH03x sensor
```

Comment:
Place this function within #ifdef PMS_MODEL_ZH03X

Reply:
Done